backend/src/database/seeds/08-Pronouns.ts

```diff
@@ -0,0 +1,14 @@
+import { Knex } from 'knex';
+
+export async function seed(knex: Knex): Promise<void> {
+  // Deletes ALL existing entries and reset primary key
+  await knex.raw('TRUNCATE TABLE pronouns RESTART IDENTITY CASCADE');
+
+  // Inserts seed entries
+  await knex('pronouns').insert([
```

Comment:
I was expecting this to be single entries. i.e. He, Her, Him, She, etc

Then the climber can select multiple (up to 4), which is how Instagram does it.

Though right now, I thought about it again, it's good enough.